src/utils.js

```diff
@@ -1,9 +1,17 @@
-import { castPath } from './key-path'
+/**
+ * @flow
+ */
```

Comment:
Wondering if you know anything about flow working together with JSDocs? Also, can we just use `// @flow` for brevity and to not confuse the JSDocs writing? Maybe that's a silly thing, but thoughts?